tasks/install-rhel.yml

```diff
@@ -5,6 +5,5 @@
 
 - name: install foreman rubygem-apipie-bindings package
   yum: name=rubygem-apipie-bindings state=present
-  when: foreman_proxy_register  
```

Comment:
Difference is that we add install-ubuntu.yml, so install-rhel.yml file name should have same naming pattern.